package.json

```diff
@@ -121,25 +126,8 @@
 					"type": "boolean",
 					"default": false
 				},
-				"metaed.telemetryConsent": {
-					"order": 5,
```

Comment:
No longer necessary. new VSCode telemetry API only works when user opts-in to VSCode's consent